### ISSUE TITLE
`<sub>`/`<sup>` within `<small>`/`.small` can result in too small of a font size

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Framework#216]
**Describe the bug**

See title; because `<sub>`/`<sup>` and `<small>`/`.small` reduce font size by a percentage, the resulting font size of nested `<sub>`/`<sup>` elements can be below 9px, which is considered a minimum readable font size for the web.

**Steps to Reproduce**

1. `<p class="small"><sub>:(</sub></p>`
2. Scan with accessibility scanner

**Expected Behavior**

Any text styles provided by Athena should be capped at a minimum of 9px rendered at standard zoom.

**Possible Fix**

Consider adjusting the `font-size` value for `<sub>`/`<sup>` and maybe `<small>`/`.small` to use rem/em's

**Your environment:**

n/a

**Additional context**

n/a
